tests/test_spd_matrices_space.py

```diff
@@ -125,6 +124,20 @@ def test_procrustes_inner_product(self):
 
         self.assertAllClose(result, expected)
 
+    @geomstats.tests.np_and_tf_only
+    def test_power_affine_inner_product(self):
+        base_point = gs.array([[1., 0., 0.],
+                               [0., 2.5, 1.5],
+                               [0., 1.5, 2.5]])
+        tangent_vec = gs.array([[2., 1., 1.],
+                               [1., .5, .5],
+                               [1., .5, .5]])
+        metric = SPDMetric(3, power_affine=.5)
```

Comment:
Let's put this line in the setup, so that we can then call self.metric in the tests.

Reply:
Line 135? But we use different metrics in different tests... I don't get your point.

Reply:
then self.affine_metric?